frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js

```diff
@@ -368,6 +368,19 @@ class DataCube {
     }
   }
 
+  setBucketData(zoomedAddress: Vector4, data: Uint8Array) {
+    const bucket = this.getOrCreateBucket(zoomedAddress);
+    if (bucket.type === "null") {
+      return;
+    }
+    bucket.setData(data);
+    this.pushQueue.insert(bucket);
+  }
+
+  triggerPushQueue() {
+    this.pushQueue.push();
```

Comment:
In other cases where pushQueue.push is called it is followed by `segmentationLayer.cube.trigger("volumeLabeled");` which let's the mapping info view know to update itself I think. You should add that here as well. While you're at it, you could replace other calls to `pushQueue.push()` followed by `trigger("volumeLabeled")` by a call to this function :)